examples/llil_parser/src/LowLevel_IL_Parser.cpp

```diff
@@ -0,0 +1,433 @@
+		}
+
+
+		operandId++;
+	}
+
+
+}
+
+void LlilParser::decodeIndexInFunction(uint64_t functionAddress, int indexIl)
+{
+
+	m_currentFunction = m_bv->GetAnalysisFunctionsForAddress(functionAddress);
+	if (m_currentFunction.size() < 1)
+		throw std::runtime_error("Error no functions at requested address\n");
+
+	BinaryNinja::Function *function = m_currentFunction[0];
+	BinaryNinja::Ref<BinaryNinja::LowLevelILFunction> llil = function->GetLowLevelIL();
+
+	m_currentInstructionId = indexIl;
+	BNLowLevelILInstruction currentInstruction = llil->operator[](llil->GetIndexForInstruction(indexIl));
+
+
+	analysisInstruction(currentInstruction);
+	m_tabs = 0;
+
+}
+
+void LlilParser::decodeWholeFunction(BinaryNinja::Function *function)
+{
+	m_currentFunction.clear();
+	m_currentFunction.push_back(function);
+
+	BinaryNinja::Ref<BinaryNinja::LowLevelILFunction> llil = function->GetLowLevelIL();
+
+	for (size_t i = 0; i < llil->GetInstructionCount(); i++)
+	{
+		
+		m_currentInstructionId = i;
+		BNLowLevelILInstruction currentInstruction = llil->operator[](llil->GetIndexForInstruction(i));
+		
+		printf("\n[%zx][%zd]---------------------------------------------------------------------------\n", currentInstruction.address, i);
+
+		analysisInstruction(currentInstruction);
+		m_tabs = 0;
+	}
+
+}
+
+void LlilParser::decodeWholeFunction(uint64_t functionAddress)
+{
+
+	m_currentFunction = m_bv->GetAnalysisFunctionsForAddress(functionAddress);
+	if (m_currentFunction.size() < 1)
+		throw std::runtime_error("Error no functions at requested address or possible invalid BundledPluginDirectory\n");
+
+	BinaryNinja::Function *function = m_currentFunction[0];
+	BinaryNinja::Ref<BinaryNinja::LowLevelILFunction> llil = function->GetLowLevelIL();
+
+	
+
+	for (size_t i = 0; i < llil->GetInstructionCount(); i++)
+	{
+		m_currentInstructionId = i;
+		BNLowLevelILInstruction currentInstruction = llil->operator[](llil->GetIndexForInstruction(i));
+		printf("\n[%zx][%zd]---------------------------------------------------------------------------\n", currentInstruction.address, i);
+
+		analysisInstruction(currentInstruction);
+		m_tabs = 0;
+	}
+
+}
+
+void ShowBanner()
+{
+
+	printf (".____    .____    .___.____      __________                                   \n");
+	printf ("|    |   |    |   |   |    |     \\______   \\_____ _______  ______ ___________ \n");
+	printf ("|    |   |    |   |   |    |      |     ___/\\__  \\\\_  __ \\/  ___// __ \\_  __ \\\n");
+	printf ("|    |___|    |___|   |    |___   |    |     / __ \\|  | \\/\\___ \\\\  ___/|  | \\/\n");
+	printf ("|_______ \\_______ \\___|_______ \\  |____|    (____  /__|  /____  >\\___  >__|   \n");
+	printf ("        \\/       \\/           \\/                 \\/           \\/     \\/    \n");
+	printf("====================================================================================\n\n");
+
+}
+
+#ifndef _WIN32
```

Comment:
seems like BinaryNinja::GetUserPluginDirectory() returns C:\Users\Development\AppData\Roaming\Binary Ninja\plugins which
is an empty directory instead of C:\Program Files\Vector35\BinaryNinja\plugins which has the required platform dlls, if i use that one i get 0 functions
inside